src/multiexp.rs

```diff
@@ -328,17 +326,7 @@ where
         assert!(query_size == exponents.len());
     }
 
-    let result = pool.compute(move || multiexp_inner(bases, density_map, exponents, c));
-    #[cfg(feature = "gpu")]
-    {
-        // Do not give the control back to the caller till the
-        // multiexp is done. We may want to reacquire the GPU again
-        // between the multiexps.
-        let result = result.wait();
-        Waiter::done(result)
-    }
-    #[cfg(not(feature = "gpu"))]
-    result
+    multiexp_inner(bases, density_map, exponents, c)
```

Comment:
I think this one needs to run in an `THREAD_POOL.install()`. Hopefully that will fix the failing CI.

Reply:
done